DSharpPlus/Entities/Channel/Message/DiscordMessageFile.cs

```diff
@@ -39,6 +39,15 @@ internal DiscordMessageFile(string fileName, Stream stream, long? resetPositionT
             this.ResetPositionTo = resetPositionTo;
         }
 
+        public DiscordMessageFile(string fileName, Stream stream, bool resetPosition = false)
+        {
+            this.FileName = fileName;
+            this.Stream = stream;
+            this.FileType = null;
+            this.ContentType = null;
```

Comment:
are we sure we want those unconditionally set to null? how are they handled in the rest APIs taking them?

Reply:
I looked through all references to DiscordMessageFile and I only saw these being used when creating stickers.